### PR TITLE
VB-3927 Booker account management - MVP

### PR DESCRIPTION
**Pull request by tpmcgowan**
Add basic booker account management to the Visits admin service:
* Search for a booker by email
* Add a new booker (by email)
* Add a prisoner to a booker
* Add visitor(s) to a booker/prisoner record
* Make prisoner and visitor entries active/inactive
* Clear booker details (wipes prisoner and visitor data, but preserves the booker email and reference)

N.B. No test coverage: this will come in a follow-on PR...

### CODE REVIEW

File: assets/scss/components/_restrictions.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.visitor-restriction {
```
Comment (tpmcgowan) [inline assets/scss/components/_restrictions.scss:1]: Copied these coloured tags for restrictions from Staff UI

File: server/utils/nunjucksSetup.ts [hunk 2]
```diff
@@ -70,5 +71,26 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
 
   njkEnv.addFilter('pluralise', (word, count, plural = `${word}s`) => (count === 1 ? word : plural))
 
+  njkEnv.addFilter('displayAge', (dateOfBirth: string) => {
```
Comment (tpmcgowan) [inline server/utils/nunjucksSetup.ts:74]: Using same filter as public and staff UIs

File: server/views/partials/visitorRestrictions.njk [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% macro visitorRestrictions(visitor) %}
```
Comment (tpmcgowan) [inline server/views/partials/visitorRestrictions.njk:3]: Macro copied from Staff UI

File: server/views/pages/bookers/bookers.njk [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+{% block content %}
+{{ super() }}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <h1 class="govuk-heading-l">{{ pageHeaderTitle }}</h1>
+
+    {% if action == "search" %}
+      <p>Search for an existing booker or a add a new one.</p>
```
Comment (simonnebMOJ) [inline server/views/pages/bookers/bookers.njk:19]: small typo needs fixing - remove errant 'a'

File: server/views/pages/bookers/booker/addPrisoner.njk [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+{% block content %}
+{{ super() }}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <h1 class="govuk-heading-l">{{ pageHeaderTitle }}</h1>
+
+    <p>Add a prisoner to booker <strong>{{ booker.email }}</strong>.</p>
```
Comment (simonnebMOJ) [inline server/views/pages/bookers/booker/addPrisoner.njk:18]: Remove final fullstop

File: server/views/pages/bookers/bookers.njk (hunk 4 above)
Comment (tpmcgowan) [reply server/views/pages/bookers/bookers.njk]: Fixed!

File: server/views/pages/bookers/booker/addPrisoner.njk (hunk 5 above)
Comment (tpmcgowan) [reply server/views/pages/bookers/booker/addPrisoner.njk]: Fixed!

**Review by hutcheonb-moj — APPROVED** [review]
Code LGTM - unless any content needs changing

**Pull request merged**